lang/en-gb.xml

```diff
@@ -36,13 +36,12 @@
 	</about>
 
 	<app_context>
-		<boot>Boot</boot>
-		<check_app_compatibility>Check App Compatibility</check_app_compatibility>
+		<check_app_state>Check App state</check_app_state>
 		<copy_vita3k_summary>Copy Vita3K Summary</copy_vita3k_summary>
-		<open_issue>Open Issue</open_issue>
-		<create_issue>Create Issue</create_issue>
-		<update_database>Update Database</update_database>
-		<copy_app_info>Copy App Info</copy_app_info>
+		<open_state_report>Open state report</open_state_report>
```

Comment:
I think `compat report` would be better.

Reply:
> I think `compat report` would be better.

no, is already compat, menu is Compatiblity/Open state report, and state is perfect